api/app/signals/apps/email_integrations/reporter/mail.py

```diff
@@ -34,11 +34,17 @@ def send_mail_reporter_created(signal):
                             recipient_list=recipient_list, html_message=html_message)
 
 
-def send_mail_reporter_status_changed_afgehandeld(signal, status):
+def send_mail_reporter_status_changed_afgehandeld(signal, status, prev_status):
     signal_is_afgehandeld = status.state == workflow.AFGEHANDELD
     if not signal_is_afgehandeld or not signal.reporter.email:
         return None
 
+    # SIG-1619 We should not send a mail after request to repoen a "melding" /
+    # Signal was closed, hence:
+    prev_status_is_verzoek_tot_heropenen = prev_status.state == workflow.VERZOEK_TOT_HEROPENEN
```

Comment:
Why store this in a value instead of just writing the if like `if prev_status.state == workflow.VERZOEK_TOT_HEROPENEN:`? Storing it in a variable will cause it to be stored in memory and we never use it after the if.

I know this is done a lot in the mail integrations when there is a check on a status. So more of a general question here.

Reply:
done